src/App.tsx

```diff
@@ -15,36 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
+  default = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  isReversed: boolean,
+) {
```

Comment:
The function name 'getPreparedGoods' should start with a verb to indicate what it does following the [CODE STYLE] checklist. Consider renaming it to something like 'prepareGoods' to more accurately describe its action.